### DISCUSSION TITLE
View source blocks

### DISCUSSION THREAD

**Discussion by olegl** [Eloston/ungoogled-chromium#1796] [General]
Just recently, commit `e72fc9b64116bf259e516096fcc60b58ae8ae1b3` [dropped](https://chromium.googlesource.com/chromium/src/+/e72fc9b64116bf259e516096fcc60b58ae8ae1b3%5E%21/) on Chromium sources, disallowing the "view source" in browser, if the website requests it to do so. There is no backlash [at Google](https://chromium-review.googlesource.com/c/chromium/src/+/3260807), but [elsewhere](https://twitter.com/jwz/status/1458521191338188800?s=20) [though](https://www.jwz.org/blog/2021/11/putting-on-my-hacker-hoodie-to-view-source/).

What is your position on this? I'd, personally, rather not have this patch in ungoogled-chromium. (And also not the devtools block.)

**Comment by kramred** [comment]
I doubt there are many places where ungoogled chromium is installed on Windows with this __enterprise policy__ enabled.
If you know any examples or you can think of a scenario why ungoogled chromium should not honour that __enterprise policy__ please elaborate.

So that would be Windows (Active Directory) clients where the admin set that policy and also installed ungoogled chomium – in the linked blog post they talk about preventing students on a school computer from using `view source`. 

Now you want ungoogled chomium not to follow the policy that the admin set who also made ungoogled chromium available?

• If I was that admin I would compile ungoogled chromium without such a patch so that it to behaves as I configured it.
• If I was a student I would be mad that they have to resort to such cheap tricks to prevent me from looking at the source code of the page (and pity the the developer who could not think of a better way than storing the answers or whatever in the page source)


My __personal__ position on this is that there's nothing to do here right now. 

See also [Objectives](https://github.com/Eloston/ungoogled-chromium#objectives) and [Motivation and Philosophy](https://github.com/Eloston/ungoogled-chromium#motivation-and-philosophy)
